setup.py

```diff
@@ -68,7 +68,6 @@ def _get_plugin_version():
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
-        "Programming Language :: Python :: 3.10",
     ],
-    python_requires=">=3.7",
+    python_requires=">=3.7,<3.10",
```

Comment:
This might be good enough without explicitly setting the upper bound. We could just say `dbt-databricks` does not **officially** support Python 3.10 yet.
@allisonwang-db WDYT?

Reply:
Yea this should be fine.